installing/acs-installation-platforms.adoc

```diff
@@ -71,7 +71,7 @@ Not all installation methods are supported for all platforms.
 
```

Comment:
(not about this line)
I see `:ocp-supported-version: 4.5` in `modules/common-attributes.adoc`.
https://github.com/openshift/openshift-docs/blob/7033db690dbf62b5a80707441a1c40e82b69e497/modules/common-attributes.adoc#L57-L57

Maybe it's unused but it should be cleaned up or updated.

Reply:
Thank you Misha. I'll create a separate PR for fixing the supported version number.

Reply:
I don't think it is optional any more.
With the latest changes I see this https://60483--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/prerequisites-ocp.html

![image](https://github.com/openshift/openshift-docs/assets/537715/4992ee6a-0641-403f-af3c-b2e3901cc840)